libc/src/unistd/linux/pathconf_utils.cpp

```diff
@@ -0,0 +1,113 @@
+//===-- Linux implementation of pathconf_utils
+//---------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/__support/OSUtil/syscall.h" // For internal syscall function.
+#include "src/__support/common.h"
+
+#include "src/errno/libc_errno.h"
+#include <sys/syscall.h> // For syscall numbers.
+
+namespace LIBC_NAMESPACE {
+
+static long filesizebits(const struct statfs &s) {
```

Comment:
`static` will make the symbol only visible to the same TU, which should be wrong here.

Reply:
Does that mean when we link the program, when the other 2 files try to make the function call in this file, they will get some error like "undefined reference" ?

Reply:
Yes